### DISCUSSION TITLE
multiple selection for custom controls

### DISCUSSION THREAD

**Discussion by pizza7311** [fabricjs/fabric.js#8498] [Q&A]
[http://fabricjs.com/custom-controls-polygon](http://fabricjs.com/custom-controls-polygon)  

i have read the this example for implementing shape modifying  
it's called 'direct selecte' in adobe illustrator  
but this example can only select one point(controller)  
i want to make this example like this  
![2 points selected](https://user-images.githubusercontent.com/51106108/206394965-d3d94ae0-81dd-4013-8b63-5c5c848da1c7.png)  

you can try this at here  
[https://w00dn.github.io/papergrapher/](https://w00dn.github.io/papergrapher/)  
draw the rectangle and switch to 'detail select' (shortcut A key) then drag to select corners  
so, is there's way to select multiple controls?  
i spent my times for implementing this feature and i think this a last way

**Comment by ShaMan123** [comment]
You can implement whatever you want in the action handler

**Reply by ShaMan123** [reply]
same goes for rendering controls
